packages/ui/src/Table/Table.tsx

```diff
@@ -2,7 +2,10 @@ import React from "react";
 
 import TableCell, { CellType } from "./TableCell";
 
-type TableItem = Record<string, string | number | boolean | null>;
+type TableItem = Record<
+  string,
+  string | number | boolean | null | { booked: number; delta: number }
+>;
```

Comment:
Normally, I would advise against this: By allowing `{ booked: number; delta: number }` as a value you're turning the `Table` (imagined as a generic component) in to an opinionated one.

Buuut, in the spirit of pragmatism, **this table is used only once** so there's no harm here - _If we had more time to devote to this, I would even advise merging the two components into AttendanceVarianceTable, but this is ok._

Reply:
I knew this would give you the ick...but yeah I was thinking the same thing